filter_multi_user/models/ir_filters.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class IrFilters(models.Model):
+    _inherit = "ir.filters"
+
+    user_ids = fields.Many2many(
+        comodel_name="res.users",
+        string="Users",
+    )
+
+    @api.model
+    def get_filters(self, model, action_id=None):
+        """WARNING: this function overrides the standard one.
```

Comment:
pls do not override the docstring. Use a comment instead.

Reply:
Fixed :)